docs/docs/channels/push/one-signal.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: One Signal
+sidebar_position: 1
+---
+
+# One Signal
+
+[OneSignal](https://onesignal.com/) is a paid push notification service that supports sending messages via both [Apple Push Notification Service](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server) (APNs) as well as [Firebase Cloud Messaging](https://firebase.google.com/docs/cloud-messaging) (FCM).
+
+To configure the OneSignal integration, you will need an active account which has credentials for APNs, FCM or both, and have access to the `OneSignal App ID` and `Rest API Key` available via your [application's settings page](https://documentation.onesignal.com/docs/keys-and-ids).
+
+## Set Device Token
+
+Before triggering the notification to a subscriber(user) with push as a step in the workflow, make sure you have added the subscriber's `userId` from OneSignal as their device token. This is also referred to as the `Player ID` within the OneSignal web console:
+
+```ts
+import { Novu, PushProviderIdEnum } from '@novu/node';
+
+const novu = new Novu(process.env.NOVU_API_KEY);
+
+await novu.subscribers.setCredentials('subscriberId', PushProviderIdEnum.OneSignal, {
+  deviceTokens: ['token1', 'token2'],
+});
```

Comment:
Could you in the code snippet represent what you are explaining to set the device token, so it is more clear for other users that have less experience with OneSignal and our platform? 🙏🏻

Reply:
Updated, cheers 👍